### DISCUSSION TITLE
new ui in min browser?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2153] [Q&A] [answered]
Hi everyone.

I was thinking of redesigning Min Browser to look like a text editor.

![image](https://user-images.githubusercontent.com/111701513/209214513-828d882e-0fd1-4a11-9046-cd9c1487ef4a.png)

**img-description** This image is illustrative with [joinpresage](https://www.joinpresage.com/) license. 

What do you all think of the idea?

**Comment by PalmerAL** [comment, marked as answer]
Can you explain what this would look like in a browser? Is the main idea to have a vertical tab bar?

**Reply by ghost** [reply]
Hi PalmerAL.

> Can you explain what this would look like in a browser?

Yes.

> Is the main idea to have a vertical tab bar?

No... would be... 2 sidebars.
